modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/wstep/impl/CertificateEnrollmentServiceImpl.java

```diff
@@ -126,101 +118,54 @@ public void requestSecurityToken(String tokenType, String requestType,
                 headerBinarySecurityToken = element.getFirstChild().getNextSibling().getFirstChild().getTextContent();
             }
         }
-
+        List<ConfigurationEntry> tenantConfigurations;
+        try {
+            if (getTenantConfigurationData() != null) {
+                tenantConfigurations = getTenantConfigurationData();
```

Comment:
you can call getTenantConfigurationData() once, assign the returned value for a local variable and check for null. Otherwise this is a recall of the same method which is redundant.